### PR TITLE
🗾 Add list hovering and sync list results on map

### LINKED ISSUE #141: Hovering over list entry in desktop shows that it's selected on the map
## Learning objectives

- Objective

## Todo

These points are a rough guideline. **Please** feel free to discuss with others on the team about the best way to design this component!

- Todo

## Useful links

- Useful link

### PR DESCRIPTION
**Pull request by Qrtn**
## Status:
:rocket: Ready

## Description

Fixes #141 

## Screenshots

image

### CODE REVIEW

File: src/components/list/EntryList.js [hunk 1]
```diff
@@ -31,7 +31,9 @@ const EntryList = forwardRef(
       itemSize,
       itemCount,
       locations,
-      handleListEntryClick,
+      handleEntryClick,
+      handleEntryMouseEnter,
```
Comment (SirajChokshi) [inline src/components/list/EntryList.js:35]: Maybe we can just call these `onClick`, `onMouseEnter`, and `onMouseLeave`. No reason to reinvent the prop interface imo.

**Review by SirajChokshi — APPROVED** [review]
Clever Girl 🦖 
tenor

File: src/components/list/EntryList.js (hunk 1 above)
Comment (Qrtn) [reply src/components/list/EntryList.js:35]: Oh crap, yeah they should be on not handle lol.

Comment (Qrtn) [reply src/components/list/EntryList.js:35]: Do you think `Entry` is necessary? Since it's for a specific item in the list.

Comment (SirajChokshi) [reply src/components/list/EntryList.js:35]: Let's keep `Entry`. That's a good distinction to make.

**Comment by ezwelty** [comment]
Don't forget to remove all locations (including those added from the list) from the map when back in cluster mode:

image

**Comment by Qrtn** [comment]
Oh shoot, thanks Ethan.

**Pull request merged**